test/functional/lua/buffer_updates_spec.lua

```diff
@@ -256,9 +256,9 @@ describe('lua: nvim_buf_attach on_bytes', function()
   -- assert the wrong thing), but masks errors with unflushed lines (as
   -- nvim_buf_get_offset forces a flush of the memline). To be safe run the
   -- test both ways.
-  local function setup_eventcheck(verify)
-    meths.buf_set_lines(0, 0, -1, true, origlines)
-    local shadow = deepcopy(origlines)
+  local function setup_eventcheck(verify, start_txt)
```

Comment:
note to self: we can allow start_txt as a multiline string and split it on `\n` automatically